frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -123,14 +136,8 @@ It is important that once you have constructed your object, that you perform ini
 #### Prediction model
 You can download models to perform prediction for the following tissues:
 
-For use in `venv` environments:
-- [HLCA Lung Model](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/hlca_20220728144320_venv.zip)
-- [Pan Tissue Immune Reference](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/immune_ref_20220728145305_venv.zip)
-
-For use in `conda` environments:
-- [HLCA Lung Model](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/hlca_20220728144320_conda.zip)
-- [Pan Tissue Immune Reference](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/immune_ref_20220728145305_conda.zip)
-
+- [HLCA Lung Model](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/hlca_20220920223732.zip)
+- [Pan Tissue Immune Reference](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/immune_ref_20220920223732.zip)
```

Comment:
removed dual py envs; updated model versions

Reply:
perfect - makes this part cleaner too :)